spec/src/main/asciidoc/chapters/introduction/entities.asciidoc

```diff
@@ -0,0 +1,142 @@
+== Type system
+
+This specification assumes that data is represented in a structured form, that is, that it is expressed in terms of:
+
+1. **atomic values**,
+2. **collections**, and
+3. **structures**.
+
+Any atomic value, collection, or structure belongs to at least one well-defined type.
+
+=== Atomic values
+
+_Atomic values_ are things defined externally to this specification, taken to include--at bare minimum--strings of Unicode characters, integers, floating point numbers, and dates, times, and datetimes.
+Every atomic value has a named type, referred to in the sister-specifications of this specification as a _basic type_.
+
+[NOTE]
+The atomic value types allowed by an implementation of Jakarta Query depend on the programming language and underlying database technology.
+
+Atomic values of distinct type are never considered identical (equal).
```

Comment:
```suggestion
Atomic values of distinct types are never considered identical (equal).
```

Reply:
I think the singular is correct here. We can say that two things are of "distinct color", or "distinct size", or "distinct type".